test/experiments_ClimaCore.jl

```diff
@@ -0,0 +1,80 @@
+# Experiments
+
+using Test
+using Pkg
+
+Pkg.activate(joinpath(@__DIR__,"../experiments/ClimaCore/tc4_atm-lnd-sfc-fluxes/"))
+
+Pkg.add(url="https://github.com/CliMA/ClimaAtmos.jl",rev="main" )
+Pkg.add(url="https://github.com/CliMA/SurfaceFluxes.jl",rev="sf_rc0.2" )
+
+Pkg.instantiate()
+Pkg.precompile()
```

Comment:
I need to look at CI setup to see if we can avoid this kind of environment activation. We could always add all this to the `test/Project.toml` I mentioned above.